index.html

```diff
@@ -0,0 +1,349 @@
+        <li class="skil button">bootstrap</li>
+        <li class="skil button">Ruby</li>
+      </ul>
+    </div>
+    <a class="see-project-2, button" href="">See Project</a>
+
+
+    <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <!-- <h2 class="desk-prof-art-dash">Data Dashboard Healthcare</h2> -->
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+        industry's standard</p>
+      <ul class="skils">
+        <li class="button">html</li>
+        <li class="skil button">bootstrap</li>
+        <li class="skil button">Ruby</li>
+      </ul>
+    </div>
+    <a class="see-project-2 button" href="">See Project</a>
+
+    <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <!-- <h2 class="desk-prof-art-web">Website Protfolio </h2> -->
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+        industry's standard</p>
+      <ul class="skils">
+        <li class="button">html</li>
+        <li class="skil button">bootstrap</li>
+        <li class="skil button">Ruby</li>
+      </ul>
+    </div>
+    <a class="see-project-2 button" href="">See Project</a>
+    <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <!-- <h2 class=" desk-prof-art">Profesional Art Printing Data</h2> -->
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+        industry's standard</p>
+      <ul class="skils">
+        <li class="button">html</li>
+        <li class="skil button">bootstrap</li>
+        <li class="skil button">Ruby</li>
+      </ul>
+    </div>
+    <a class="see-project-2 button" href="">See Project</a>
+    <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <!-- <h2 class="desk-prof-art-dash">Data Dashboard Healthcare</h2> -->
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+        industry's standard</p>
+      <ul class="skils">
+        <li class="button">html</li>
+        <li class="skil button">bootstrap</li>
+        <li class="skil button">Ruby</li>
+      </ul>
+    </div>
+    <a class="see-project-2 button" href="">See Project</a>
+    <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <!-- <h2 class="desk-prof-art-web">Website Protfolio </h2> -->
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+        industry's standard</p>
+      <ul class="skils">
+        <li class="button">html</li>
+        <li class="skil button">bootstrap</li>
+        <li class="skil button">Ruby</li>
+      </ul>
+    </div>
+    <a class="see-project-2 button" href="">See Project</a>
```

Comment:
Kindly take note of the `see project` button for the first card😉 let's give it more `width` so it looks more like the design 😉😉😉

yours
<img width="905" alt="yoursssssssssssssssssssssssssssssssssssssssssssssssssssssssss" src="https://user-images.githubusercontent.com/64643995/220949704-802b19dc-0033-497a-92fc-7f6e2f2819bb.png">


design
<img width="473" alt="project card" src="https://user-images.githubusercontent.com/64643995/220949209-44633648-916c-4011-9365-d7ab5bed5a73.png">